themes/default/content/blog/enhanced-search-intuitive-navigation-new-docs-experience/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Enhanced Search Intuitive Navigation New Docs Experience"
+date: 2023-06-12
+meta_desc:  At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully.
+meta_image: meta.png
+authors:
+    - susan-evans
+    - anita-trimbur
+    - sean-holung
+tags:
+    - pulumi-releases
+    - features
+    - ux
+---
+
+Engineers spend a lot of time searching documentation for answers. At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully. Today, we announce an improved Pulumi documentation experience dedicated to all developers.
+<!--more-->
+
+We prioritized search and navigation, so you can easily browse and search for precisely what you want. Enjoy faster and more accurate searches, with lovely user experience touches to guide you to the right place. We also planned for the future, using design patterns that will grow with content and concepts.
+
+![Pulumi docs welcome banner](imgs/banner.png)
+
+Let's showcase a few types of Pulumi documentation:
+
+## Quickstart experience
+
+The [Pulumi quickstart](/docs/quickstart/) experience is for people new to Pulumi to get up and running quickly.
+
+![Pulumi docs quickstart experience](imgs/quickstart.png)
+
+## Cloud get started tutorials
+
+The [Pulumi cloud get started tutorials](/docs/clouds/) are for people who want a detailed walkthrough the first time they use Pulumi.
+
+![Pulumi cloud get started tutorials](imgs/get-started.png)
+
+## Concept docs
+
+The [Pulumi concept docs](/docs/concepts/) are where you can dive into Pulumi concepts, how they work together, and how to effectively use them to ship infrastructure.
+
+![Pulumi concept docs](imgs/concepts.png)
+
+## Packages
+
+The [package documentation](/docs/registry/) is where to find config and API documentation for Pulumi supported clouds, version control systems, databases, & infrastructure, monitoring, and networking tools. We support 119 providers today, and we add new providers every week!
```

Comment:
> clouds, version control systems, databases, & infrastructure, monitoring, and networking tools

I think I know why it's written this way ("clouds, version control systems, databases" are one group, "infrastructure, monitoring, and networking tools" are another group) but it reads awkwardly. We may want to consider re-wording it.

Reply:
@sean1588 fixed it! 💜